packages/crypto/test/IncrementalMerkleTree.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai'
+import { genRandomNumber, IncrementalMerkleTree } from '../src'
+
+describe('IncrementalMerkleTree', function () {
+    let tree: IncrementalMerkleTree
+    const depth = 4
+
+    it('constructor', async () => {
```

Comment:
We use the `zk-kit` implementation for this, we shouldn't test it ourselves. Instead our higher level tests should fail if the tree implementation is broken.